app/models/task.rb

```diff
@@ -9,15 +9,20 @@ class Task < ApplicationRecord
 
   acts_as_tenant(:tenant)
 
+  @@timeout_interval = 120 * 60 # 2 hours
+
+  def timeout_interval
```

Comment:
@hsong-rh Should this be a class method

Reply:
No, I use a simpler way to set `timeout_interval`